### ISSUE TITLE
The Mortuary Assistant (1295920)

### ISSUE THREAD

**Issue by diegoviola** [ValveSoftware/Proton#6157]
# Compatibility Report
- Name of the game with compatibility issues: The Mortuary Assistant
- Steam AppID of the game: 1295920

## System Information
- GPU: Ryzen 5 2400G integrated graphics
- Driver/LLVM version: mesa 22.1.7-1
- Kernel version: 5.19.7-arch1-1
- Proton version: Proton 7.0-4

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 

The game starts fine after starting a new shift, I see the introductory video and the game starts. However, when I restart the game and click on continue (to continue the existing gameplay) I get a loading screen and it hangs right there (I still get sound when I get the hang).

Here's the log: [steam-1295920.log](https://github.com/ValveSoftware/Proton/files/9571423/steam-1295920.log)

A workaround I found (to avoid the hang) is to start the game at a lower resolution (for example: 1440x900) with low image quality.

## Reproduction

- Start a new game
- Restart the game and click on Continue.
- You should get a loading screen where it hangs.

**Comment by ndarlington** [comment]
I also tried with Proton 7.0-5 Release Candidate per #6240 just in case it helped.  It does not.

**Comment by ndarlington** [comment]
Unsure if it's some kind of file corruption, or a timing issue, but if I delete the /home/deck/.steam/steam/steamapps/shadercache/[steam_appid] folder and play the game again, it works again for one more time.

So whenever the game starts up and the shader cache files are present, it seems to hang/fail.

**Comment by Blisto91** [comment]
@ndarlington Hello. 
Could you try launching with either `PROTON_NO_FSYNC=1 %command%` or `PROTON_NO_FSYNC=1 PROTON_NO_ESYNC=1 %command%` and see if that makes a difference?

**Comment by ndarlington** [comment]
Apologies for the delayed response.  I went to see about applying this, but Proton release 7.05 was no longer a release candidate and had been pushed out, my device was updated with it.  Since then, the game has run without issue even without the workaround I was using.

Additionally, the game has been submitted for review to Valve and has since earned a green 'verified' checkmark.  So this issue should be resolved.  Thank you.
